modules/extension/wms/src/main/java/org/geotools/data/wms/WMS1_1_0.java

```diff
@@ -212,47 +203,12 @@ public InternalGetLegendGraphicRequest( URL onlineResource, WMS1_1_0 creator) {
         protected void initVersion() {
             setProperty(VERSION, "1.1.0");
         }
-        
-        public Response createResponse(HTTPResponse httpResponse) throws ServiceException, IOException {
-			return new GetLegendGraphicResponse(httpResponse);
-		}	    
-	}
-	
-	public static class InternalGetStylesRequest extends AbstractGetStylesRequest {
```

Comment:
I am not sure I understand did you remove GetStyles support from WMS 1.1.0 ... in order to add it to WMS 1.1.1?